subsys/net/lib/azure_iot_hub/include/azure_iot_hub_mqtt.h

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef AZURE_IOT_HUB_MQTT__
+#define AZURE_IOT_HUB_MQTT__
+
+#include <stdio.h>
+#include <net/azure_iot_hub.h>
+#include <net/mqtt.h>
+#include <net/socket.h>
```

Comment:
Is this inclusion really needed in this header? I don't see any use of its contents.

Reply:
Doesn't seem so, removed.